src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -6,6 +6,18 @@ object InputView {
         val input = readln()
         return input.split(",").map { it.trim() }.filter { it.isNotEmpty() }
     }
+
+    fun getBettingAmount(playerName: String): Int {
+        println("Enter $playerName’s betting amount (multiple of 1000):")
+        val input = readLine()?.toIntOrNull()
+        return if (input != null && input % 1000 == 0 && input > 0) {
+            input
+        } else {
+            println("Invalid amount. Please enter a positive number that is a multiple of 1000.")
```

Comment:
It's hard to generate test code. right?

I think u can create the `Bet` value class, and append validation, and generate test code more easier!